ramls/customField.json

```diff
@@ -4,15 +4,21 @@
   "description": "Collection of custom fields",
   "additionalProperties": false,
   "properties": {
-    "name": {
+      "id": {
```

Comment:
minor nitpick on the spacing here for "id"

Reply:
@sduvvuri-ebsco done